TradeItIosTicketSDK2/TradeItYahooLauncher.swift

```diff
@@ -53,7 +53,7 @@ import SafariServices
         )
     }
 
-    public func handleOAuthCallback(
+    @objc public func handleOAuthCallback(
         onTopmostViewController topMostViewController: UIViewController,
         oAuthCallbackUrl: URL,
         onOAuthCompletionSuccessHandler: OnOAuthCompletionSuccessHandler? = nil
```

Comment:
Verify this works. Same situation/comment as `onViewPortfolioTappedHandler` basically.

Reply:
@blanksblanks did you verify this?